sympy/printing/llvmjitcode.py

```diff
@@ -136,6 +136,74 @@ def _print_Symbol(self, s):
         return value
 
 
+class LoopCreator(object):
```

Comment:
Any public classes and methods will need at least doc strings.

I'd love to see your example added to the Sphinx documentation too.